api-generation/src/main/scala/ai/h2o/sparkling/api/generation/Runner.scala

```diff
@@ -142,7 +145,8 @@ object Runner {
       ("H2OGLRM", classOf[GLRMParameters], "H2OUnsupervisedAlgorithm", Seq.empty),
       ("H2OGAM", classOf[GAMParameters], "H2OSupervisedAlgorithm", Seq.empty),
       ("H2OPCA", classOf[PCAParameters], "H2OUnsupervisedAlgorithm", Seq.empty),
-      ("H2OIsolationForest", classOf[IsolationForestParameters], "H2OTreeBasedUnsupervisedAlgorithm", Seq.empty))
+      ("H2OIsolationForest", classOf[IsolationForestParameters], "H2OTreeBasedUnsupervisedAlgorithm", Seq.empty),
+      ("H2OWord2Vec", classOf[Word2VecParameters], "H2OUnsupervisedAlgorithm", Seq("H2OWord2VecExtras")))
```

Comment:
Is `H2OWord2Vec` an unsupervised H2O algorithm? Shouldn't it belong to the same category and namespace as `H2OTargetEncoder`?

Reply:
H2OWord2Vec is unsupervised algorithm. But can move it

Reply:
If I'm not mistaken Word2Vec is treated in H2O as an algorithm since there is no concept of transformers and estimators. IMHO, Word2Vec is a feature engineering stage that has to be combined with another algorithm. Spark categorizes as a feature estimator as well: https://spark.apache.org/docs/latest/ml-features.html#word2vec

cc @michalkurka

Reply:
ok

Reply:
I can change it, but would first merge PR in this state and iterate after in another PR

Reply:
done, w2v in in the features namespace

Reply:
@mn-mikke @jakubhava great makes sense